docs/exploring-es-data.asciidoc

```diff
@@ -21,51 +22,42 @@ Default APM `template` and `indices`:
 * <<span-indices>>
 * <<error-indices>>
 * <<sourcemap-indices>>
-* <<healthcheck-indices>>
-
 
-For querying all APM data:
-["source","sh"]
-------------------------------------------------------------
-GET apm*/_search
-------------------------------------------------------------
-// CONSOLE
 
-Querying documents that have been collected with a specific APM Server version:
+To query all documents collected with a specific APM Server version:
 ["source","sh",subs="attributes"]
 ------------------------------------------------------------
 GET apm-{version}-*/_search
 ------------------------------------------------------------
 // CONSOLE
 
-If you are only interested in specific document types, e.g. error documents you can use the type in your query:
+To query a specific type, e.g. transactions:
```

Comment:
I'd use "for example" rather than "e.g." because non native English speakers might not know what e.g. means.

Reply:
e.g. is a latin voice, so all romanic languages speakers know it, and i think germanic and slavic speakers  know it too...
there are ofc more languages than those, so if you think is an actual issue i'm happy to change it. we have a lot of e.gs spread all over the place.